lib/nimble.phx.gen.template/addons/variants/web/wallaby.ex

```diff
@@ -0,0 +1,263 @@
+defmodule Nimble.Phx.Gen.Template.Addons.Web.Wallaby do
+  use Nimble.Phx.Gen.Template.Addon
+
+  @versions %{
+    wallaby: "~> 0.26.2"
+  }
+
+  @impl true
+  def do_apply(%Project{} = project, _opts) do
+    project
+    |> copy_files
+    |> edit_files
+  end
+
+  defp copy_files(%Project{} = project) do
+    binding = [
+      web_module: project.web_module,
+      base_module: project.base_module
+    ]
+
+    files = [
+      {:eex, "test/support/feature_case.ex.eex", "test/support/feature_case.ex"},
+      {:eex, "test/features/home_page/view_home_page_test.exs.eex",
+       "#{project.web_test_path}/features/home_page/view_home_page_test.exs"}
+    ]
+
+    Generator.copy_file(files, binding)
+
+    project
+  end
+
+  defp edit_files(%Project{} = project) do
+    project
+    |> inject_mix_dependency
+    |> edit_mix
+    |> edit_test_helper
+    |> edit_endpoint
+    |> edit_test_config
+    |> edit_gitignore
+    |> edit_assets_package
+
+    if File.exists?(Path.join([".github", "workflows", "test.yml"])) do
+      edit_github_action(project)
```

Comment:
Not sure if it's worth to do all these edits but instead just have a separate Github addon for web and API 💭 My main concern is maintenance

Reply:
Separate file definitely easy to maintain then, I will change that